run-queries.sh

```diff
@@ -89,12 +86,12 @@ for i in $(seq "${page_count}"); do
     #    don't care about for the same reasons.
     grep "        \"html_url" < "${curl_output_file}" \
         | grep -v "          " \
-        | sort | uniq \
+        | sort -u \
         | cut -d \" -f 4 \
         | grep -v "kelloggm" \
```

Comment:
Rather than filtering out organizations/owners here, I suggest adding "-org:kelloggm" and "-org:AndroidSDKSources" to the query.  Doing so will mean that more of the 1000 results contain useful repositories.  I also suggest adding "language:java" and "extension:java", for similar reasons.